docs/fast_data/configuration/config_maps/erSchema.md

```diff
@@ -167,6 +171,140 @@ Example:
+
+:::tip
+You can also create a condition starting and ending to the same projection, by dragging a line from the right handle and releasing near the left handle.
+:::
+
+When a line is created, you can click on it: you will see that the condition line and the two related projection boxes will change color, and on the right side panel you will be able to configure the condition.
+
+![Condition panel opened](../../img/er-schema-condition-panel-empty.png)
+
+This panel will allow you to configure a connection **both ways**. For example, having the image above as a reference, a condition will be created from _pr_dishes_ to _pr_reviews_ and vice versa, so you will not have to worry about creating another line for the opposite condition.
+
+In this panel, you can see the connected projections, referred to as **A** and **B**. In the example shown in the image above, _A_ is the projection _pr_dishes_ and _B_ is the projection _pr_reviews_.
+
+You can select the _condition type_, for both the condition and its reverse, which can be one of the two values:
+- _One to one_ (default), which means that the condition will fetch one document from a projection from one document to the other projection matching the condition;
+- _One to many_, which means that the condition will fetch all the documents from one projection that match the condition with the document in the other projection;
+You can select the _condition type_ going from _A_ to _B_ (in this example: from _pr_dishes_ to _pr_reviews_) and from _B_ to _A_ (from _pr_reviews_ to _pr_dishes_). 
+
+Below are listed the rules that define conditions both ways. These rules are divided into three sections, each one with its tab:
+- _A <-> B_: includes all those rules that will be used in the condition from _A_ to _B_ and vice versa. In the example of the screenshot above, the rule `id_dish = ID_DISH` means that the condition from _pr_dishes_ to _pr_reviews_ will connect the `id_dish` field in _pr_dishes_ to the `ID_DISH` in _pr_reviews_, and the condition from _pr_reviews_ to _pr_dishes_ will connect the `ID_DISH` in _pr_reviews_ to the `id_dish` field in _pr_dishes_;
+- _A -> B_: includes all those rules that will be used in conditions where filtering documents of projection _A_, executing queries based on comparison operations on fields of projection _B_ or constant values;
+- _B -> A_: includes all those rules that will be used only in the conditions where filtering starting from projection _B_, as explained above;
+
+:::info
+Defining conditions in both ways is highly suggested to make sure that your ER Schema can be used in any Fast Data microservice (you can read more in the [Direction of the relationships](#direction-of-the-relationships) section). The panel will allow you to do that.
```

Comment:
Do we want to say "highly suggested" instead of Mandatory? I'm not sure the Fast Data is working without the conditions declared both ways

Reply:
In my mind, not exactly. You might want to have different ER Schemas between Single View Creator and Real Time Updater since they uses different conditions, as example. But I wouldn't really use _mandatory_ here.

Reply:
sounds reasonable. Do you think it is clear for the user that different er schemas can be created?

Reply:
Maybe we should add a sentence clarifying that the ER Schema could be different for each microservice that uses it and it is not mandatory to have the same er schema for all the microservices involved in the generation of the single view

Reply:
I've updated the info box. Still not sure because I really don't feel the need to explain this concept, which seems very obvious to me. But feel free to suggest a better sentence/paragraph if you have something specific in mind.